package.json

```diff
@@ -51,33 +51,32 @@
     "homepage": "https://github.com/tinkoff/angular-open-source-starter#README",
     "schematics": "./schematics/collection.json",
     "dependencies": {
-        "@angular/animations": "^9.1.13",
-        "@angular/common": "^9.1.13",
-        "@angular/compiler": "^9.1.13",
-        "@angular/core": "^9.1.13",
-        "@angular/forms": "^9.1.13",
-        "@angular/platform-browser": "^9.1.13",
-        "@angular/platform-browser-dynamic": "^9.1.13",
-        "@angular/platform-server": "^9.1.13",
-        "@angular/router": "^9.1.13",
+        "@angular/animations": "^11.2.14",
+        "@angular/common": "^11.2.14",
+        "@angular/compiler": "^11.2.14",
+        "@angular/core": "^11.2.14",
+        "@angular/forms": "^11.2.14",
+        "@angular/platform-browser": "^11.2.14",
+        "@angular/platform-browser-dynamic": "^11.2.14",
+        "@angular/platform-server": "^11.2.14",
+        "@angular/router": "^11.2.14",
         "@ng-web-apis/common": "^1.9.0",
         "@ng-web-apis/universal": "^1.9.2",
         "@nguniversal/express-engine": "^9.1.1",
```

Comment:
When updating Angular you should also update Universal. Not sure if it can be just bumped, it could be that some packages were added/removed between 9 and 12 and some changes had to be done in `server.ts`. You can probably create clean Angular project and add universal to it to check what changes that makes.

Reply:
fixed